### PR TITLE
Form validation in change password form

### LINKED ISSUE #470: Submit button should be enabled when all fields are filled
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in our [Gitter channel](https://gitter.im/fossasia/susi_server)

**Current behavior:**
validation

**Expected behavior:**
When all fields are filled then only submit should work.

```
Your operating system: Windows
```

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #470

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
validation

#### Changes proposed in this pull request:
When all fields are valid then only submit button will be enabled.

### CODE REVIEW

File: src/js/login.js [hunk 1]
```diff
@@ -43,14 +43,26 @@ pass.addEventListener("click", ()=>{
     pass.innerHTML = ($("#cPass").toggle().is(":visible"))?("Click here to hide change password form"):("Click here to change password");
 });
 
+var password_validation = () => {
+    if (confirmPassword.value === newPassword.value && newPassword.value.length >6 && newPassword.value.length<64 && cpassword.value.length>0) {
```
Comment (neeraj3029) [inline src/js/login.js]: Change it to `newPassword.value.length >=6 && newPassword.value.length<=64`

Comment (samagragupta) [reply src/js/login.js]: @neeraj3029 Done the required changes.

**Comment by samagragupta** [comment]
@fragm3 @neeraj3029 Please review.

File: src/js/login.js [hunk 2]
```diff
@@ -43,14 +43,26 @@ pass.addEventListener("click", ()=>{
     pass.innerHTML = ($("#cPass").toggle().is(":visible"))?("Click here to hide change password form"):("Click here to change password");
 });
 
+let password_validation = () => {
```
[the remark this answers is not in the document]
Comment (samagragupta) [reply src/js/login.js]: @fragm3 Done the changes

**Comment by samagragupta** [comment]
@fragm3 Done.

**Comment by samagragupta** [comment]
@hkedia321 Please review.

**Comment by samagragupta** [comment]
@Orbiter @stealthanthrax

**Pull request merged**